tests/mythril/mythril_analyzer_test.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+from mythril.mythril import MythrilDisassembler, MythrilAnalyzer
+
+
+def test_fire_lasers():
```

Comment:
what specifically is this test aimed at?

Reply:
It tests `MythrilDisassembler.fire_lasers()`

Reply:
I think we should limit the number of components this test is hitting, maybe some mocks?